elasticsearch.md

```diff
@@ -1759,6 +1765,37 @@ group_search_filter: "(cn=*)" # basically no group filtering
 
 Caching can be configured per LDAP client \(see `ldap1`\) or per rule \(see `Accept requests from users in group team2 on index2` rule\)
 
+#### LDAP Server discovery
+
+It is possible for LDAP module to get all LDAP hostnames from DNS server rather than from configuration file. By default `_ldap._tcp` SRV records are used for that, but any other SRV record can be configured.
+
+The simplest configuration of LDAP using server discovery is:
+```
+    - name: ldap
+      server_discovery: true                                        
+      search_user_base_DN: "ou=People,dc=example2,dc=com"
+      search_groups_base_DN: "ou=Groups,dc=example2,dc=com"
+```
+It is using system DNS to get `_ldap._tcp` SRV records which should contain hostnames and ports to connect to.
```

Comment:
... which are expected to contain the hostname and port of all the LDAP servers we should connect to.